examples/nil-lifting/nil_lifting.description

```diff
@@ -1,4 +1,4 @@
-// The previous example demonstrates how to eliminate null reference errors by using optional types in Ballerina.
+// The `Optional Type` example demonstrates how to eliminate null reference errors by using optional types in Ballerina.
 // This example demonstrates how to operate on optional typed variables.
```

Comment:
Add more about the use case and why this is useful. Also shall we describe what a lifted operator is here so that it's understood when used in the code?